File: dayTwo/python/level4/level_four.py
def add(first_number, second_number):
    return first_number + second_number

def checkNumber(number):
    if number %2 == 0:
        print("Even")
    else:
        print("Odd")
def square(number):
    return number**2
    
def celsiusToFahrenheit(celsius):
    return (celsius* (9/5)) + 32

def largestNumber(firstNumber, secondNumber, thirdNumber):
    largest = firstNumber
    if(secondNumber > firstNumber):
        largest = secondNumber
        
    elif (thirdNumber > secondNumber):
        
        largest = thirdNumber
        
        print(largest)
        
    
    
    
    
    
    
    
    
    
      
    
print(add(2,3))
checkNumber(3)
print(square(3))
print(celsiusToFahrenheit(3))
largestNumber(1,2,3)

File: dayTwo/java/level4/LevelFour.java
public class LevelFour{

    public static void main(String[] args){
   
   System.out.println(add(2,4));
   checkEvenNumber(4);
   System.out.println(square(4));
   System.out.println(celsiusToFahrenheit(4));
   System.out.println(areaOfRectangle(5,2));
   System.out.println(simpleInterest(5000,2.5,2));
   largestNumber(5000,2,2);
   
   
   
   

    }

    public static int add(int firstNumber, int secondNumber){
    
         return  firstNumber + secondNumber;
    }
    
     public static void  checkEvenNumber(int number){
     
     if(number %2 == 0){
       System.out.println("Even");
       }
     else{
     System.out.print("Odd");
     }
      
    }
    public static int square(int number){
    
        return number * number;
    }
    
    public static double celsiusToFahrenheit(double celsius){
    
        return (celsius*(9/5)) + 32;
    }
   
    public static double areaOfRectangle(double width, double length){
    
        return width * length;
    }
     public static double simpleInterest(double principal, double rate, double time){
    
        return (principal * rate * time)/100;
    }
    public static void largestNumber(int firstNumber, int secondNumber, int thirdNumber){
        int largest = firstNumber;
        if(secondNumber > firstNumber){
        largest = secondNumber;
        }
        else if (thirdNumber > secondNumber){
        largest = thirdNumber;
        }
        System.out.println(largest);
        }
}


























